clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -97,7 +97,7 @@ spec:
         - --ci-operator-config-path=/etc/configs
         - --github-endpoint=http://ghproxy
         - --github-token-path=/etc/github/oauth
-        - --testImagesDistributorOptions.imagePullSecretPath=/etc/regcred/.dockerconfigjson
+        - --testImagesDistributorOptions.imagePullSecretPath=/etc/registry-pull-credentials/.dockerconfigjson
```

Comment:
Please leave this out. The first PR should only make sure the new secret is created and available. Do not make anything use it yet.